tools/depends/configure.ac

```diff
@@ -437,11 +471,21 @@ case $use_platform in
   raspberry-pi3)
      target_platform=raspberry-pi
      use_cpu=cortex-a53
+     use_cmake_cpu=armv7-a # Raspberry mostly only use 32bit, also on the newer versions with 64bit CPU.
      ffmpeg_options_default="--cpu=cortex-a53"
      platform_cflags="-fPIC -mcpu=cortex-a53 -mfloat-abi=hard -mfpu=neon-fp-armv8 -mvectorize-with-neon-quad"
      platform_cxxflags="-fPIC -mcpu=cortex-a53 -mfloat-abi=hard -mfpu=neon-fp-armv8 -mvectorize-with-neon-quad"
      platform_ldflags="-lpthread"
      ;;
+  raspberry-pi4)
+     target_platform=raspberry-pi
+     use_cpu=cortex-a72
+     use_cmake_cpu=armv7-a # Raspberry mostly only use 32bit, also on the newer versions with 64bit CPU.
+     ffmpeg_options_default="--cpu=cortex-a72"
+     platform_cflags="-fPIC -mcpu=cortex-a72 -mfloat-abi=hard -mfpu=neon-fp-armv8 -mvectorize-with-neon-quad"
+     platform_cxxflags="-fPIC -mcpu=cortex-a72 -mfloat-abi=hard -mfpu=neon-fp-armv8 -mvectorize-with-neon-quad"
+     platform_ldflags="-lpthread"
+     ;;
```

Comment:
How is adding a new Linux platform related to this PR?
Have you runtime tested it?

Reply:
Was wrong added, only thought for test at home. Becomes removed.